packages/Sandblocks-Core/SBEditor.class.st

```diff
@@ -1487,6 +1509,18 @@ SBEditor >> startOrAddToMultiSelection: aBlock [
 	self cursor startOrAddToMultiSelection: aBlock
 ]
 
+{ #category : #stepping }
+SBEditor >> step [
+
+	self repositionSuggestionsMenu
```

Comment:
could we move this to the suggestions menu itself? idea would be that we then only step when we actually need to, without doing bookkeeping ourselves. we could then also store the reference as inst var on the menu instead of a property.

Reply:
Building on this: I like the addition of SBEditor>>showSuggestionsMenu:near:, if the above change is implemented I would suggest to only move #respositionSuggestionsMenu to the menu class.

Reply:
Sounds good, I will address this in the next days :)